benchmarks/test_emu.py

```diff
@@ -73,3 +73,47 @@ def test_emu(model):
     pk = ccl.nonlin_matter_power(cosmo, k, a)
     err = np.abs(pk/data[:, 1]-1)
     assert np.allclose(err, 0, rtol=0, atol=EMU_TOLERANCE)
+
+
+@pytest.mark.parametrize('model', list(range(3)))
+def test_emu_lin(model):
+    cosmos = np.loadtxt("./benchmarks/data/emu_input_cosmologies.txt")
+
+    mnu = ccl.nu_masses(
+        cosmos[model, 7] * cosmos[model, 2]**2, 'equal', T_CMB=2.725)
+
+    cosmo = ccl.Cosmology(
+        Omega_c=cosmos[model, 0],
+        Omega_b=cosmos[model, 1],
+        h=cosmos[model, 2],
+        sigma8=cosmos[model, 3],
+        n_s=cosmos[model, 4],
+        w0=cosmos[model, 5],
+        wa=cosmos[model, 6],
+        m_nu=mnu,
+        m_nu_type='list',
+        Neff=3.04,
+        Omega_g=0,
+        Omega_k=0,
+        transfer_function='boltzmann_camb',
+        matter_power_spectrum='emu',
+    )
+
+    a = 1
+    k = np.logspace(-3, -2, 50)
+
+    # Catch warning about neutrino linear growth
+    if (np.sum(mnu) > 0):
+        pk = ccl.pyutils.assert_warns(ccl.CCLWarning,
+                                      ccl.nonlin_matter_power,
+                                      cosmo, k, a)
+    else:
+        pk = ccl.nonlin_matter_power(cosmo, k, a)
```

Comment:
Small thing - should we use ccl.linear_matter_power here? Same above in 108

Reply:
I'm not sure what you mean: these two lines are intended to get the non-linear matter power from the emulator for the comparison, and the linear_matter_power for the comparison is in line 114.

Reply:
Ah, ok, sorry I was thinking of the line 114 input indeed